persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -44,7 +44,8 @@ CREATE TABLE Conversation (
     verification_status TEXT AS ConversationEntity.VerificationStatus NOT NULL DEFAULT "NOT_VERIFIED",
     proteus_verification_status TEXT AS ConversationEntity.VerificationStatus NOT NULL DEFAULT "NOT_VERIFIED",
     degraded_conversation_notified INTEGER AS Boolean NOT NULL DEFAULT 1,
-    legal_hold_status TEXT AS ConversationEntity.LegalHoldStatus NOT NULL DEFAULT "DISABLED"
+    legal_hold_status TEXT AS ConversationEntity.LegalHoldStatus NOT NULL DEFAULT "DISABLED",
+    legal_hold_status_change_notified INTEGER AS Boolean NOT NULL DEFAULT 1
```

Comment:
`legal_hold_status_change_notified` is not a data that belongs to conversation entity, it's more like a preference. I think it would be better if we move it to `SharedPrefrence` or maybe `Metadata` table

Reply:
I just did it exactly the same way as it's already implemented for informing the user about degraded conversation before sending a message, it's very similar case and as you can see there is already also a column `degraded_conversation_notified`, it got approved and I would like to keep the code consistent.

Reply:
question: maybe the better approach is to have a separated table with a foreign key conversation id of conversation that need to be notified ?

Reply:
Ok, I created a separate table for that flag.